lib/validation.js

```diff
@@ -277,6 +277,28 @@ module.exports = function (yargs, usage, y18n) {
     }
   }
 
+  var conflicting = {}
+  self.conflicts = function (key1, key2) {
+    if (typeof key1 === 'string' && typeof key2 === 'string' && key1 !== '_' && key2 !== '_') {
```

Comment:
That looks weird... i don't know why I did this last part of the test... to me it looks like some leftover (I tried some approaches and ended up with the simplest one)